src/components/visit/formField.tsx

```diff
@@ -0,0 +1,42 @@
+interface FormFieldProps {
+  id: string
+  label: string
+  type: string
```

Comment:
Minor: You could improve this type by specifying exactly which strings are valid, maybe something like `type: 'text' | 'number' | 'textarea'` etc.